packages/erxes-ui/src/components/richTextEditor/extensions/Table/Header/Header.ts

```diff
@@ -6,6 +6,9 @@ export const Header = TiptapHeader.extend({
       style: {
         parseHTML: (element) => element.getAttribute('style'),
       },
+      align: {
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **🚨 issue (security):** Ensure that the 'align' attribute is properly handled in the table header extension.